### PR TITLE
chore(flutter): add snapcraft.yaml example in platform setup

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
_Issue #, if available:_

_Description of changes:_
- Add an example of how to package dependencies on linux

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by dnys1 — APPROVED** [review]

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx [hunk 1]
```diff
@@ -100,6 +100,23 @@ sudo apt-get install -y libglib2.0-dev
 
 The command to install might vary slightly on other Linux distributions.
 
+### Packaging Your App
+To include the dependencies above when packaging your app with Snapscraft, include them in your snapcraft.yaml file.
```
Comment (dnys1) [inline src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx:104]:
```suggestion
To include the dependencies above when packaging your app with Snapcraft, include them in your `snapcraft.yaml` file.
```

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx [hunk 2]
```diff
@@ -100,6 +100,23 @@ sudo apt-get install -y libglib2.0-dev
...
+  my-app:
+    plugin: flutter
+    source: .
+    flutter-target: lib/main.dart
+    build-packages:
+      - libsecret-1-dev
+      - libglib2.0-dev
+    stage-packages:
+      - libsecret-1-dev
+      - libglib2.0-dev
```
[the remark this answers is not in the document]
Comment (Jordan-Nelson) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx:117]: Yeah, I wasn't sure about this tbh so I tried to test it. I was having trouble testing this, but I think it was just general snapcraft flakiness on a VM. I'll attempt to test this again.

[the remark this answers is not in the document]
Comment (katiegoines) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx:117]: @Jordan-Nelson - Wanted to check in on whether this has been resolved. Thank!

[the remark this answers is not in the document]
Comment (Jordan-Nelson) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx]: Apologies for the long delay. I went ahead and made this update. I am still having trouble doing a full test (packaging w/ snapcraft and running), but from the logs I can see it is pulling the dep correctly when this was updated.

**Comment by Jordan-Nelson** [comment]
@abdallahshaban557 - Can you review for the PM team?
@offlineprogrammer - Can you review for the support-eng team?
@dnys1 - Can you review for amplify-flutter?
@rachelnabors - Can you review for the documentation team?

Thanks

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx [hunk 3]
```diff
@@ -110,7 +110,7 @@ Add a development team in the "Signing and Capabilities" section and enable sign
 No specific configuration steps are needed to begin using the Auth category on Windows; however, depending on how your Windows configuration and the method you use to run your Flutter app, you may need to change the file path length limitation.
 
 ## Linux
-To run the Amplify-Flutter developer preview on Linux, you must install two libraries:
+Amplify-Flutter developer preview depends on libsecret and glib when targeting Linux. To run and debug an app using Amplify-Flutter, you must install the following libraries:
```
Comment (abdallahshaban557) [inline src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx:113]: Can we remove the dashes between Amplify Flutter please?

**Review by abdallahshaban557 — COMMENTED** [review]
Left one comment

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx (hunk 3 above)
Comment (abdallahshaban557) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx:113]: @Jordan-Nelson

**Review by dnys1 — APPROVED** [review]
LGTM

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx (hunk 3 above)
Comment (Jordan-Nelson) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx]: @abdallahshaban557 updated 👍

File: src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx [hunk 4]
```diff
@@ -100,6 +100,23 @@ sudo apt-get install -y libglib2.0-dev
 
 The command to install might vary slightly on other Linux distributions.
 
+### Packaging Your App
+To include the dependencies above when packaging your app with Snapcraft, include them in your `snapcraft.yaml` file.
```
[the remark this answers is not in the document]
Comment (Jordan-Nelson) [reply src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx]: Updated

**Review by offlineprogrammer — APPROVED** [review]

**Review by abdallahshaban557 — APPROVED** [review]
LGTM!

**Pull request merged**